test/src/api/commands/web-element/testIsVisible.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert');
+const {WebElement} = require('selenium-webdriver');
+const MockServer  = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+describe('element().isVisible() command', function() {
+  before(function (done) {
+    CommandGlobals.beforeEach.call(this, done);
+  });
+
+  after(function (done) {
+    CommandGlobals.afterEach.call(this, done);
+  });
+
+  it('test .element().isVisible() Visible', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/Visible',
+      method: 'GET',
+      response: JSON.stringify({
+        value: true
+      })
+    }, true);
+
+    this.client.api.element('#search').isVisible(function (result) {
+      this.assert.equal(result.value, true);
+    });
+  });
```

Comment:
These tests are wrong:
1. '/session/13521-10219-202/element/0/Visible' endpoint does not exist.
2. `isVisible()` command does not accept a callback.

The only reason this test is passing is because no assertion is being run in actual, because all the assertions are present in the callback which is never called.

It seems to me that you've just blindly copied the tests from some other PR and didn't care to understand what it is doing or ask questions.